src/lib/server/db/reports.channels.posts.sql.ts

```diff
@@ -0,0 +1,23 @@
+import { pgTable, text, uuid } from 'drizzle-orm/pg-core';
+import { post } from './posts.sql';
+import { channel } from './channels.sql';
+
+export const channelPostReport = pgTable('channel_post_report', {
+    id: uuid('id').primaryKey().defaultRandom(),
+    postId: uuid('post_id')
+        .notNull()
+        .references(() => post.id),
+    channelId: uuid('channel_id')
+        .notNull()
+        .references(() => channel.id),
```

Comment:
This is technically redundant as the post already has a reference to its channel and they should always be the same

Reply:
Totally right. Removing